app/models/concerns/individual_market_behaviors.rb

```diff
@@ -0,0 +1,15 @@
+module IndividualMarketBehaviors
+  extend ActiveSupport::Concern
+
+  included do
+    def individual_market_is_enabled?
+      @individual_market_is_enabled ||= Settings.aca.market_kinds.include? "individual"
```

Comment:
I realized I can just delegate this method to the class - I'll fix it when I do the merge.